app/src/androidTest/java/org/greenstand/android/TreeTracker/database/AppDatabaseTest.kt

```diff
@@ -0,0 +1,53 @@
+package org.greenstand.android.TreeTracker.database
+
+import android.content.Context
+import androidx.room.Room
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import junit.framework.TestCase
+import kotlinx.coroutines.runBlocking
+import org.greenstand.android.TreeTracker.database.legacy.entity.PlanterInfoEntity
+import org.hamcrest.CoreMatchers.*;
+import org.junit.After
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+
+@RunWith(AndroidJUnit4::class)
+class AppDatabaseTest: TestCase(){
```

Comment:
The TreTrackerDAO has had a test added to it already. Feel free to delete this and add any extra test cases to the already created test file.